### PR TITLE
library/functions

### PR DESCRIPTION
**Pull request by Ardasak**
Translated till 637th line.

### CODE REVIEW

**Review by egeakman — DISMISSED** [review]

File: copyright.po [hunk 1]
```diff
@@ -53,4 +53,5 @@ msgstr ""
 msgid ""
 "See :ref:`history-and-license` for complete license and permissions "
 "information."
-msgstr "Tam lisans ve izin bilgileri için :ref:`geçmiş ve lisans` 'a göz atın."
+msgstr ""
+"Tam lisans ve izin bilgileri için :ref:`history-and-license` 'a göz atın."
```
Comment (Ardasak) [inline copyright.po:57]:
```suggestion
"Bütün lisans ve izin bilgileri için :ref:`history-and-license` 'a göz atın."
```

**Review by Ardasak — COMMENTED** [review]
reviewing my own work

File: library/functions.po [hunk 2]
```diff
@@ -504,46 +553,61 @@ msgid ""
...
 
 #: library/functions.rst:184
 msgid ""
 "If it is a *string*, you must also give the *encoding* (and optionally, "
 "*errors*) parameters; :func:`bytearray` then converts the string to bytes "
 "using :meth:`str.encode`."
 msgstr ""
+"Eğer bir *string* ise, *encoding* (ve istenirse, *errors*) parametrelerini "
+"de vermelisiniz; sonrasında :func:`bytearray` :meth:`str.encode` kullanarak "
+"string'i bayta çevirecektir."
```
Comment (Ardasak) [inline library/functions.po:578]:
```suggestion
"dizeyi bayta çevirecektir."
```

File: library/functions.po [hunk 3]
```diff
@@ -552,19 +616,27 @@ msgid ""
...
+"0 <= x < 256 aralığındaki tam sayılardan oluşan değiştirilemez bir bayt "
+"objesi döndürür. :class:`bytes`, :class:`bytearray` 'in değiştirilemez "
+"versiyonudur. Aynı objeyi değiştirmeyen metotlara, indeksleme ve dilimleme "
+"davranışına sahiptir."
 
 #: library/functions.rst:211
 msgid ""
 "Accordingly, constructor arguments are interpreted as for :func:`bytearray`."
-msgstr ""
+msgstr "Buna göre, yapıcı argümanlar :func:`bytearray` için yorumlanır."
```
Comment (Ardasak) [inline library/functions.po:627]:
```suggestion
msgstr "Buna göre, yapıcı argümanları :func:`bytearray` için yorumlanır."
```

File: library/functions.po [hunk 4]
```diff
@@ -574,42 +646,62 @@ msgid ""
...
+"Bu fonksiyon Python 3.0 versiyonunda kaldırılmıştı ama Python 3.2 "
+"versiyonunda geri getirildi."
 
 #: library/functions.rst:233
 msgid ""
 "Return the string representing a character whose Unicode code point is the "
 "integer *i*.  For example, ``chr(97)`` returns the string ``'a'``, while "
 "``chr(8364)`` returns the string ``'€'``. This is the inverse of :func:`ord`."
 msgstr ""
+"Unicode kod noktası *i* tamsayısı olan bir karakteri temsil eden string "
```
Comment (Ardasak) [inline library/functions.po:670]:
```suggestion
"Unicode kod noktası *i* tamsayısı olan bir karakteri temsil eden dizeyi "
```

File: library/functions.po [hunk 5]
```diff
@@ -574,42 +646,62 @@ msgid ""
...
+"döndürür. Örneğin, ``chr(97)`` , ``'a'`` string'ini döndürürken, "
```
Comment (Ardasak) [inline library/functions.po:671]:
```suggestion
"döndürür. Örneğin, ``chr(97)`` , ``'a'`` dizesini döndürürken, "
```

File: library/functions.po [hunk 6]
```diff
@@ -574,42 +646,62 @@ msgid ""
...
+"``chr(8364)`` , ``'€'`` string'ini döndürür. Bu fonksiyon, :func:`ord` 'un "
```
Comment (Ardasak) [inline library/functions.po:672]:
```suggestion
"``chr(8364)`` , ``'€'`` dizesini döndürür. Bu fonksiyon, :func:`ord` 'un "
```

File: library/functions.po [hunk 7]
```diff
@@ -574,42 +646,62 @@ msgid ""
...
+msgstr "Bir metodu sınıf metoduna dönüştürür."
 
 #: library/functions.rst:245
 msgid ""
 "A class method receives the class as an implicit first argument, just like "
 "an instance method receives the instance. To declare a class method, use "
 "this idiom::"
 msgstr ""
+"Bir sınıf metodu, sınıfı örtük ilk argüman olarak alır, aynı sınıftan "
+"türeyen bir örneğin metodunun örneği aldığı gibi. Bu sınıf metodunu bu "
```
Comment (Ardasak) [inline library/functions.po:695]:
```suggestion
"türeyen bir örneğin metodunun örneği aldığı gibi. Bir sınıf metodunu bu "
```

File: library/functions.po [hunk 8]
```diff
@@ -646,13 +750,18 @@ msgid ""
 "string, a byte string, or an AST object.  Refer to the :mod:`ast` module "
 "documentation for information on how to work with AST objects."
 msgstr ""
+"*source* 'ı bir koda veya AST nesnesine derler. Kod nesneleri :func:`exec` "
+"veya :func:`eval` fonksiyonları kullanılarak çalıştırılabilir."
```
Comment (Ardasak) [inline library/functions.po:754]:
```suggestion
"*source* 'ı bir koda veya AST nesnesine derler. Kod nesneleri :func:`exec` "
"veya :func:`eval` fonksiyonları kullanılarak çalıştırılabilir. "
"*source* normal bir dize, bayt dizesi veya bir AST nesnesi olabilir. "
"AST nesneleriyle nasıl çalışılacağını öğrenmek için :mod:`ast` modülüne bkz."
```

**Review by Ardasak — COMMENTED** [review]
reviewing my own work, will continue from 800

File: library/functions.po [hunk 9]
```diff
@@ -676,6 +793,16 @@ msgid ""
 "*flags* argument is it -- the flags (future features and compiler options) "
 "in the surrounding code are ignored."
 msgstr ""
+"*flags* ve *dont_inherit* opsiyonel argümanları hangi :ref:`derleyici "
+"seçenekleri <ast-compiler-flags>` 'ın aktive edilmesi gerektiğini ve hangi :"
+"ref:`gelecekteki özellikler <future>` 'a izin verileceğini kontrol eder. "
```
Comment (Ardasak) [inline library/functions.po:798]:
```suggestion
"ref:`gelecekteki özellikler <future>` 'e izin verileceğini kontrol eder. "
```

File: library/functions.po [hunk 10]
```diff
@@ -676,6 +793,16 @@ msgid ""
...
+"İkisi de verilmediyse (veya ikisi de sıfır ise) kod, :func:`compile` 'ı "
+"çağıran kodu etkileyen aynı bayraklarla derlenir. Eğer *flags* argümanı "
+"verilip *dont_inherit* verilmediyse (veya sıfırsa), derleyici seçenekleri ve "
+"gelecek ifadeleri *flags* argümanı tarafından belirtilirler, yine de "
```
Comment (Ardasak) [inline library/functions.po:802]:
```suggestion
"gelecek ifadeleri *flags* argümanı tarafından belirtilirler, her türlü"
```

File: library/functions.po [hunk 11]
```diff
@@ -773,27 +942,37 @@ msgid ""
...
+"geri döner. ``__float__()`` tanımlanmadıysa, :meth:`__index__` 'e geri döner."
 
 #: library/functions.rst:372
 msgid ""
 "When converting from a string, the string must not contain whitespace around "
 "the central ``+`` or ``-`` operator.  For example, ``complex('1+2j')`` is "
 "fine, but ``complex('1 + 2j')`` raises :exc:`ValueError`."
 msgstr ""
+"Bir dize dönüştürürken,  dizenin, merkezi `` + '' veya `` -`` operatörünün "
+"etrafında boşluk içermesine gerek yoktur. Örnek olarak, ``complex('1+2j')`` "
```
Comment (Ardasak) [inline library/functions.po:956]:
```suggestion
"etrafında boşluk içermemelidir. Örnek olarak, ``complex('1+2j')`` "
```

File: library/functions.po [hunk 12]
```diff
@@ -802,25 +981,37 @@ msgid ""
 "function deletes the named attribute, provided the object allows it.  For "
 "example, ``delattr(x, 'foobar')`` is equivalent to ``del x.foobar``."
 msgstr ""
+":func:`setattr` fonksiyonuna yakındır. Argümanlar bir nesne ve bir dizedir. "
+"Dize nesnenin özelliklerinden birinin adı olmak zorundadır. Fonksiyon "
+"nesnenin izin vermesi koşuluyla, dizede belirtilen bu özelliği siler. Örnek "
+"olarak, ``delattr(x, 'foobar')`` , ``del x.foobar`` 'a eşittir."
```
Comment (Ardasak) [inline library/functions.po:987]:
```suggestion
"olarak, ``delattr(x, 'foobar')`` , ``del x.foobar`` 'a eşdeğerdir."
```

File: library/functions.po [hunk 13]
```diff
@@ -3,427 +3,444 @@
...
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2021-12-17 23:20+0300\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
-"Language: \n"
+"PO-Revision-Date: 2022-07-10 23:37+0300\n"
+"Last-Translator: \n"
+"Language-Team: TURKISH\n"
+"Language: tr_TR\n"
```
Comment (egeakman) [inline library/functions.po:14]:
```suggestion
"Language: tr\n"
```

**Review by egeakman — CHANGES_REQUESTED** [review]

**Review by egeakman — APPROVED** [review]

**Pull request merged**